### DISCUSSION TITLE
AdvancedExtruderGenerator skipping radial nodes?

### DISCUSSION THREAD

**Discussion by Spudman105** [idaholab/moose#27347] [Q&A Meshing]
I am trying to make a pipe in moose and have made the circle in 2D using the annularmeshgenerator. However when I use the AdvancedExtruderGenerator to extrude, I lose some of the rings of nodes that are in the original circle. I have tried a bunch of the other commands and feel like I'm missing something?
```
[Mesh]
  [face1]
      type = AnnularMeshGenerator
      rmin = 0 # makes plane a disk instead of a ring
      rmax = .5 # sets the radius from the center for the outer ring, r=.5 means diameter is 1 which is beam spot size.
      nt = 10 # number of notes along outer ring 
      nr = 3 # number of nodes between the center and the outer radius. (increase later but small now for speed of tests)
  []
  
  [extrude]
      type = AdvancedExtruderGenerator
      input = face1
      heights = '1 2'
      num_layers = '1 1 '
      direction = '0 0 1'
      biases = '1 2'
  []
[]
```

![image](https://github.com/idaholab/moose/assets/26098674/1302af83-7a79-4764-a1df-96ff4c278000)

**Comment by maxnezdyur** [comment]
Can you check that by clipping the mesh in the middle? I would do more checks to confirm that these elements and nodes are missing.

**Comment by GiudGiud** [comment]
hello

This is just wireframe not showing the inner nodes.
use `surface with edges` instead in the selector (center top of the screen, it defaults to surface usually)
